### DISCUSSION TITLE
PrimeReact: Polymorphic Components

### DISCUSSION THREAD

**Discussion by imnash21** [primefaces/community#2409] [PrimeReact]
Please make all components a polymorphic, like:

```
<Button as={Link} label="..." />

<Card as="section" ...>
    ...
</Card>

...etc
```

Or

```
<Button component={Link} label="..." />

<Card component="section" ...>
    ...
</Card>

...etc
```

**Comment by nitrogenous** [comment]
Thank you for your feedback. We are currently working on the new version of PrimeReact, and we will take your suggestion into consideration. If you need any further assistance, please feel free to reopen this issue
